packages/unlock-js/src/web3Service.ts

```diff
@@ -899,26 +899,6 @@ export default class Web3Service extends UnlockService {
     return await unlockContract.publicLockLatestVersion()
   }
 
-  /**
-   * Returns referrer fees
-   */
-  async referrerFees({
-    lockAddress,
-    network,
-    address,
-  }: {
-    lockAddress: string
-    network: number
-    address: string
-  }) {
-    const lockContract = await this.getLockContract(
-      lockAddress,
-      this.providerForNetwork(network)
-    )
-    const referrerFees = await lockContract.referrerFees(address)
-    return ethers.BigNumber.from(referrerFees).toNumber()
-  }
```

Comment:
No longer needed since we don't use this anymore. This calls the referrerFees from smart contract, but in our case we use subgraph 👍

Reply:
I would keep this since someone may be depending on this downstream. The library is published and used by others.

Reply:
+1

Reply:
Good call 👍. Adding it back 🙏